pages/architecture.rst

```diff
@@ -12,7 +12,7 @@ The image below is a first attempt to identify all the required components for m
+At first glance, the different components are divided into four different *spaces*. This is done to distinguish between different levels of trust and (probably) different non-functional requirements like scaling. *Vendor space* has the highest level of trust. It includes both personal and medical data. Any components required to make Nuts work **must** be implemented by the vendor. *Service space* is also a trusted space since it'll include decrypted personal data, but not medical data. Since it consists of quite a few components a vendor can buy this as part of a service package from a different vendor. For example, a simple PGO might want get a subscription to a SaaS service which includes both *Service space* and *Nuts space* which only requires the PGO to implement the *Patient callback*. *Service space* also handles the administration of all the certificate logic for TLS connections. *Nuts space* only contains encrypted personal data and decrypted personal data that has already been made public. The primary goal of *Nuts space* is to make sure all required data for running the Nuts network is distributed across all nodes without any centrally controlled component. This notion of different service levels allows smaller vendors to connect to Nuts and at the same time provide an opportunity for the current network providers to develop new business. *Nuts foundation* is a separate entity which has the responsibility for adding the nodes. Essentially it controls the network. Although we don't like centrally controlled components, with the current state of technology, a central authority or root is required. In this case the *Nuts foundation* will control the root certificate.
```

Comment:
The claim:
> with the current state of technology, a central authority or root is required

might start some discussions with the B-word as possible solution. Is it really the case that we need a central authority or is this the most pragmatic solution for the moment/stack? It deserves some more explainations since Nuts is boasting its decentralized architecture and this says otherwise.